### ISSUE TITLE
Fails to setup pyvex on NixOS

### ISSUE THREAD

**Issue by Pamplemousse** [angr/angr-dev#85]
After I `nix-shell`, I use `./setup.py -p venv`, which triggers the following error after some time:

```bash
[...]
ERROR: Command "/home/pamplemousse/.virtualenvs/venv/bin/pypy3 -c 'import setuptools, tokenize;__file__='"'"'/home/pamplemousse/Workspace/angr/pyvex/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' develop --no-deps" failed with error code 1 in /home/pamplemousse/Workspace/angr/pyvex/
Exception information:
Traceback (most recent call last):
  File "/home/pamplemousse/.virtualenvs/venv/site-packages/pip/_internal/cli/base_command.py", line 178, in main
    status = self.run(options, args)
  File "/home/pamplemousse/.virtualenvs/venv/site-packages/pip/_internal/commands/install.py", line 414, in run
    use_user_site=options.use_user_site,
  File "/home/pamplemousse/.virtualenvs/venv/site-packages/pip/_internal/req/__init__.py", line 58, in install_given_reqs
    **kwargs
  File "/home/pamplemousse/.virtualenvs/venv/site-packages/pip/_internal/req/req_install.py", line 910, in install
    install_options, global_options, prefix=prefix,
  File "/home/pamplemousse/.virtualenvs/venv/site-packages/pip/_internal/req/req_install.py", line 766, in install_editable
    cwd=self.setup_py_dir,
  File "/home/pamplemousse/.virtualenvs/venv/site-packages/pip/_internal/utils/misc.py", line 776, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip._internal.exceptions.InstallationError: Command "/home/pamplemousse/.virtualenvs/venv/bin/pypy3 -c 'import setuptools, tokenize;__file__='"'"'/home/pamplemousse/Workspace/angr/pyvex/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' develop --no-deps" failed with error code 1 in /home/pamplemousse/Workspace/angr/pyvex/
```

Here are the version of some components (inside the `nix-shell`):
```bash
$ nixos-version
19.03.173068.9ec7625cee5 (Koi)
$ pip --version
pip 19.1.1 from /home/pamplemousse/.virtualenvs/venv/site-packages/pip
$ python3 --version
Python 3.5.3 (fdd60ed87e941677e8ea11acf9f1819466521bf2, Jun 23 2019, 18:51:05)
[PyPy 6.0.0 with GCC 7.4.0]
$ gcc --version
gcc (GCC) 7.4.0
```

**Comment by rhelmot** [comment]
@artemist this one's for you

**Comment by Pamplemousse** [comment]
Here are the different notes and warnings that are displayed before the error:

x-special/nautilus-clipboard
copy
file:///home/pamplemousse/log.gz

**Comment by Pamplemousse** [comment]
Here are the different notes and warnings displayed before the `ERROR`: [log.gz](https://github.com/angr/angr-dev/files/3374991/log.gz) .

**Comment by Pamplemousse** [comment]
I started contributing to Nixos/nixpkgs: I opened [several PR for `angr` components](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3APamplemousse+archived%3Afalse+org%3ANixOS+angr) .

For the record, there is an opened "feature request" to make `angr` available on Nix: NixOS/nixpkgs#67413 .

Because `pyvex` seems to build fine using the package expression I have, I am closing this issue.
